app/_src/policies/meshtimeout.md

```diff
@@ -290,6 +334,7 @@ spec:
 | http.streamIdleTimeout     | 30m     |
 | http.maxStreamDuration     | 0s      |
 | http.maxConnectionDuration | 0s      |
+{% if_version inline:true gte:2.6.x %}| http.requestHeadersTimeout | 0s      |{% endif_version %}{% if_version lte:2.5.x inline:true %} {% endif_version %}
```

Comment:
`{% if_version lte:2.5.x inline:true %} {% endif_version %}` why do we need this? 🤔

Reply:
True, my bad. We don't need it 😅